python/ray/data/grouped_data.py

```diff
@@ -218,10 +216,8 @@ def group_fn(batch, *args, **kwargs):
                 # (e.g. block is Arrow format, and batch is NumPy format).
                 group_batch = group_block_accessor.to_batch_format(batch_format)
                 applied = fn(group_batch, *args, **kwargs)
-                builder.add_batch(applied)
+                yield applied
```

Comment:
Do we worry about it's a batch format (numpy), not a block format?

Reply:
this should work the same way as a normal `map_batches` UDF, which can yield batch directly .

Reply:
yes. SG.